event-schema/event_schema.go

```diff
@@ -618,6 +636,27 @@ func getPrivateDataJSON(schemaHash string) []byte {
 }
 
 func (manager *EventSchemaManagerT) flushEventSchemas() {
+	// This will run forever. If you want to quit in between, change it to ticker and call stop()
+	// Otherwise the ticker won't be GC'ed
+	ticker := time.Tick(flushInterval)
+	for range ticker {
```

Comment:
Can we propagate a context here please? If it's too much then you can always provide a better way to stop this. a `for/select` should do the trick here.